pkg/kp/rcstore/consul_store.go

```diff
@@ -184,7 +184,7 @@ func (s *consulStore) WatchNew(quit <-chan struct{}) (<-chan []fields.RC, <-chan
 	inCh := make(chan api.KVPairs)
 
 	outCh, errCh := publishLatestRCs(inCh, quit)
-	go consulutil.WatchPrefix(rcTree+"/", s.kv, inCh, quit, errCh, 0)
+	go consulutil.WatchPrefix(rcTree+"/", s.kv, inCh, quit, errCh, 0) // TODO
```

Comment:
you could include it as an argument to `WatchNew`, I actually think 2.5M is enough to drive high bandwidth usage given the RC tree triggers many times per RU